server/channels/app/imports/import_validators.go

```diff
@@ -200,7 +200,7 @@ func ValidateUserImportData(data *UserImportData) *model.AppError {
 
 	if data.Username == nil {
 		return model.NewAppError("BulkImport", "app.import.validate_user_import_data.username_missing.error", nil, "", http.StatusBadRequest)
-	} else if !model.IsValidUsername(*data.Username) {
+	} else if !model.IsValidUsername(strings.ToLower(*data.Username)) { // we already lowercase the username while saving and querying so we are more forgiving here
```

Comment:
Could we use `model.NormalizeUser()` for consistency?

Reply:
Sure 👍